### PR TITLE
Untappd

### PR DESCRIPTION
**Pull request by Nizzlay**
Added function to use untappd social media account

### CODE REVIEW

**Comment by mtn** [comment]
Okay, you probably want to rebase this now and then you'll be good to go.

File: layouts/partials/meta.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <meta name="HandheldFriendly" content="True">
 <meta name="MobileOptimized" content="320">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<meta name="referrer" content="no-referrer">
+<meta name="referrer" content="unsafe-url">
```
Comment (mtn) [inline layouts/partials/meta.html]: Is this related to untappd?

Comment (Nizzlay) [reply layouts/partials/meta.html]: No, not required for untappd.

Comment (mtn) [reply layouts/partials/meta.html]: Sorry, I realize now that this was resolved by #109 anyways.

**Comment by mtn** [comment]
@Nizzlay Sorry for taking so long to merge this, and thanks for your contributions!
